src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<!-- Header goes here -->
```

Comment:
We just need to add the www-page component added to get the header and footer and all other global elements of the Ui app.

Reply:
I removed  header and footer comments